### DISCUSSION TITLE
Cyprus RIK HD in 1080p

### DISCUSSION THREAD

**Discussion by ROBGagn** [iptv-org/community#965] [Q&A]
Hello,

Anybody who can make auto-updated token based links for this channel from http://cybc.com.cy website in order to get the full HD (1080p) version ?

The only working link for this channel is in 720p (http://l6.cloudskep.com/tvb6/rikhd1/mpeg.2ts)

**Comment by freearhey** [comment]
If the request is still relevant, please repost it using this [form](https://github.com/iptv-org/iptv/issues/new?assignees=&labels=channel+search&projects=&template=4_channel-search.yml&title=Find%3A+).
